### PR TITLE
Style docs page

### PR DESCRIPTION
**Pull request by scottianstewart**
- Mirror spectacle code and inline code syntax highlighting
- Adjust font sizes to match

Screen Shot 2020-01-29 at 3 12 37 PM
Screen Shot 2020-01-29 at 3 12 49 PM
Screen Shot 2020-01-29 at 3 12 59 PM

### CODE REVIEW

**Comment by boygirl** [comment]
I don't _think_ the owl icons are staying since they got dropped from the logo. That would be confusing

**Comment by boygirl** [comment]
oh, never mind, I guess those owls are only on the about page. I do like having some visual indication that the headings are linkable, but i'll double check with Joe

File: src/styles/prism.js [hunk 1]
```diff
@@ -58,14 +59,23 @@ const prism = css`
...
 
     /* Inline code */
     :not(pre) > code[class*="language-"] {
-      padding: .1em;
-      border-radius: .3em;
+      opacity: 0.8;
+      font-size: 1.5rem;
+      border-width: 1px;
+      border-style: solid;
+      border-color: lightgrey;
```
[the remark this answers is not in the document]
Comment (scottianstewart) [reply src/styles/prism.js]: It seems to be a one off, but it couldn't hurt!

[the remark this answers is not in the document]
Comment (scottianstewart) [reply src/styles/prism.js]: combined

File: src/partials/playground/index.js [hunk 2]
```diff
@@ -48,58 +53,52 @@ class Playground extends React.Component {
...
-          this.content = content;
-        }}
-        dangerouslySetInnerHTML={{ __html: html }}
-      />
-    );
-  }
-}
+  return (
+    <Wrapper
+      key="content"
```
[the remark this answers is not in the document]
Comment (scottianstewart) [reply src/partials/playground/index.js]: I'm curious if we even need this key?

[the remark this answers is not in the document]
Comment (BrittanyIRL) [reply src/partials/playground/index.js]: per our slack convo, this key actually seems like it's not getting used. i vote delete!

**Pull request merged**